get.html

```diff
@@ -128,3 +128,73 @@
   <li>Let us help you get the most out of Sandstorm
   </li><a href="mailto:contact@sandstorm.io">E-mail us</a>
 </div>
+
+<div id="oasis-pricing" class="overlay">
+  <div class="popup"><a class="close" href="#">&times;</a>
+    <div class="content">
+      <section id="managed-hosting">
+       <h1>Sandstorm on Oasis: Hosting by Sandstorm.io</h1>
+       <h2>Which plan is right for you?</h2>
+       <ul class="oasis-plans">
+           <li class="oasis-plan">
+             <ul id="free">
+               <li class="title"><strong>Free</strong>
+                   <br>Start here</li>
+               <li class="price"><strong>$0</strong>
+                   <br>/forever</li>
+               <li class="storage"><strong>200MB</strong>
+                   <br>storage</li>
+               <li class="grains">5 grains</li>
+               <li class="get"><a href="https://oasis.sandstorm.io/">Get Free</a></li>
+             </ul>
+           </li>
+           <li class="oasis-plan">
+             <ul id="basic">
+               <li class="title"><strong>Basic</strong>
+                   <br>Just what you need</li>
+               <li class="price"><strong>$9</strong>
+                   <br>/month</li>
+               <li class="storage"><strong>10GB</strong>
+                   <br>storage</li>
+               <li class="grains">Unlimited grains</li>
+               <li class="get"><a href="https://oasis.sandstorm.io/">Get Standard</a></li>
+             </ul>
+           </li>
+           <li class="oasis-plan">
+             <ul id="power">
+               <li class="title"><strong>Power User</strong>
+                   <br>Boost your productivity</li>
+               <li class="price"><strong>$24</strong>
+                   <br>/month</li>
+               <li class="storage"><strong>100GB</strong>
+                   <br>storage</li>
+               <li class="grains">Unlimited grains</li>
+                 <li class="get"><a href="https://oasis.sandstorm.io/">Get Power-User</a></li>
+             </ul>
+           </li>
+        </ul>
+        
+        <div id="faq">
+        <h2>Details</h2>
+        <hr>
+          <ul>
+          <li><strong>All about grains</strong>

```

Comment:
Arguably this is some strange HTML. https://html5.validator.nu/ agrees with me that "Error: Element p not allowed as child of element ul in this context."

Why make this a UL at all? If you are hoping for semantic HTML, then it seems like "All about grains" is a heading, perhaps a H3 and can be styled accordingly. If you want just basic HTML that does the job, making them all paragraphs seems fine to me.

I'm going to declare this not-a-showstopper for now.